go/pkg/reader/reader_test.go

```diff
@@ -165,28 +165,41 @@ func TestCompressedReader(t *testing.T) {
 
 			r, err := NewCompressedFileSeeker(path, 10)
 			if err != nil {
-				t.Fatalf("Failed to initialize compressor reader: %v", err)
+				t.Fatalf("Failed to create compressor reader: %v", err)
 			}
+
+			if _, err := r.Read(make([]byte, 5)); err == nil {
```

Comment:
this test is in the same package as the implementation; i think you can make a stronger assertion here that the error is `errNotInitialized`.

Reply:
Done.